webpack.prod.js

```diff
@@ -7,6 +7,17 @@ const SSGPlugin = require('static-site-generator-webpack-plugin');
 
 // Load Common Configuration
 const common = require('./webpack.common.js');
+const Content = require('./src/_content.json');
```

Comment:
I'm starting to understand the approach now. You can't do it this way because `_content.json` only exists after a build has occurred, meaning if you haven't run a build, the file won't exist and therefore this line will fail. You should take a similar approach, but do it all within the `server.jsx` file.

Reply:
This way you won't have to change any configuration or touch any other parts of code. Then, if you remove the misc formatting things, this should be a pretty isolated, mergable PR (at least in my opinion).

Reply:
i think we could also .gitignore the _content.json? why is it in repo

Reply:
@skipjack I suspected as much, will move to `server.jsx` file

Reply:
i think moving this part:
```
const paths = Content.children.reduce((paths, page) => {
  if (page.type === 'directory') {
    page.children.forEach(child => (paths[child.url] = child.title));
  } else {
    paths[page.url] = page.title;
  }

  return paths;
}, {});

...
let title = paths[locals.path];
```

To server JSX is actually a good idea, also i just tried

```
new SSGPlugin({
        crawl: true,
        paths: [
          '/',
          '/configuration/'
        ],
        ....
      })
```
and it seems to work, but for some reason it also crawls outgoing urls from backers